application/libraries/Api/Command/V1/SurveyResponses/ResponseMappingTrait.php

```
@@ -53,6 +56,8 @@ function ($item) {
      */
     protected function mapResponsesToQuestions(array $responses, array $surveyQuestions): array
     {
+        $sodium = \Yii::app()->sodium;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$sodium'. (undefined)

(UnusedLocalVariable)